src/releasers/java-yoshi.ts

```diff
@@ -50,10 +47,19 @@ const CHANGELOG_SECTIONS = [
 ];
 
 export class JavaYoshi extends ReleasePR {
+  private versionsManifestContent?: GitHubFileContents;
+
+  protected async getVersionManifestContent(): Promise<GitHubFileContents> {
+    if (!this.versionsManifestContent) {
+      this.versionsManifestContent = await this.gh.getFileContents(
+        'versions.txt'
```

Comment:
maybe add a TODO to consolidate with @joeldodge79's manifest format eventually, if we can figure out an approach that works for both needs?